### PR TITLE
Fix Apollo Server usage

### PR DESCRIPTION
**Pull request by KATT**
#13

### CODE REVIEW

File: src/deflate.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // @flow
 
+import isPlainObject from './isPlainObject';
```
Comment (gajus) [inline src/deflate.js]: I would rather use `is-plain-object` here

https://www.npmjs.com/package/is-plain-object

Comment (KATT) [reply src/deflate.js]: Really? Your package now has zero deps! It's a rare feat and something I think should be preserved when possible. :)

Comment (gajus) [reply src/deflate.js]: What is the value in having no dependencies vs duplicating code logic across multiple packages?

Comment (KATT) [reply src/deflate.js]: 🤷‍♂️

I've gotten a bit more reserved on slapping packages in my project when I don't need to. LeftPadGate, having control and that.

Comment (KATT) [reply src/deflate.js]: But I'll change to `is-plain-object`, sec.

**Comment by KATT** [comment]
Added the npm lock-file too, always a good idea when having external deps

**Comment by gajus** [comment]
> Added the npm lock-file too, always a good idea when having external deps

Please don't. npm lock file and yarn lock file are ridiculously misunderstood. See:

https://github.com/npm/npm/issues/20603

**Comment by KATT** [comment]
Interesting subject. Rolled back!

**Pull request merged**